### PR TITLE
Update to rerun failed tests page

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description

Preview is here: https://internal.circleci.com/docs/rerun-failed-tests/

* New quickstart section to lay out what is required to use the feature, not assuming you're already using `circleci tests split`
* Update existing quickstart section to be a migration guide to moving from circleci tests split to circleci tests run
* Some updates to formatting to help readability
* Update screenshot for new UI

# Reasons
Customer feedback that we need to provide more about how to use the feature, not just specific examples, so people can more easily reconfigure test jobs to use feature

UI screenshot updates

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/rerun-failed-tests.adoc [hunk 1]
```diff
@@ -10,72 +10,126 @@ contentTags:
...
+If your current job is using xref:test-splitting-tutorial#[intelligent test splitting], you must *change* the `circleci tests split` command to `circleci tests run` combined with a `--split-by=` flag.
 
 [#quickstart]
 == Quickstart
 
+To use the rerun failed tests feature, run your tests using the `circleci tests run` command. The basic structure of your testing command should be:
+
+* Create a directory for your results
+* Glob your test files
+* Make your test files available at standard input (stdin)
```
Comment (sebastian-lerner) [inline jekyll/_cci2/rerun-failed-tests.adoc:46]: "as standard input" i think. instead of "at"

**Comment by sebastian-lerner** [comment]
small thing but can we move "knapsack" down in the list of examples? it feels odd that it's the first thing we show

**Review by sebastian-lerner — COMMENTED** [review]
looks good, one small change requested

**Comment by rosieyohannan** [comment]
> small thing but can we move "knapsack" down in the list of examples? it feels odd that it's the first thing we show

I alphabetised the examples so all Ruby examples are towards the end now

**Pull request merged**